apps/web/pages/api/user/avatar.ts

```diff
@@ -26,6 +28,9 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
       email: user?.email,
       avatar: user?.avatar,
     };
+    linksToThisRoute =
```

Comment:
This is to fix existing avatar's that are doing infinite redirects(because of the bug due to `getUserFromSession` and makes the logic more stable.

Reply:
@hariombalhara what exactly was the bug from getUserFromSession?